packages/strapi-plugin-content-manager/admin/src/lifecycles/didGetSecuredData.js

```diff
@@ -0,0 +1,24 @@
+const { map, omit } = require('lodash');
+const request = require('utils/request').default;
+const pluginId = require('../pluginId');
+
+
+module.exports = async function didGetSecuredData() {
+  const { updatePlugin } = this.props;
+  const requestURL = `/${pluginId}/models`;
+
+  try {
+    const { models: { models } } = await request(requestURL, { method: 'GET' });
+    const menu = [{
+      name: 'Content Types',
+      links: map(omit(models, 'plugins'), (model, key) => ({
```

Comment:
Why do you use the Lodash map method?

Reply:
I didn't redo the entire algo since it's going to change when the models will be retrieved by the admin container. See the comment about the `securedData: {}` in the reducer.